### PR TITLE
Top bar component

### PR DESCRIPTION
**Pull request by Athanaze**
This PR aims to fix inconsistencies regarding the top bar

### CODE REVIEW

**Comment by Athanaze** [comment]
image

**Comment by Athanaze** [comment]
This PR aims to fix inconstencies regarding the top bar.
Here are some befores:

image
image

image

**Comment by Athanaze** [comment]
Account settings before:

image

**Comment by Athanaze** [comment]
The top bar is now beautifully consistent 😎 

https://github.com/Chimpagne/ChimpagneApp/assets/29517376/93e34979-dc7a-4b15-9e11-c9c0c1d63b07

**Comment by Athanaze** [comment]
image
Fixed the alignment for this screen

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -163,6 +169,8 @@
     <string name="events_near_you">Evénements proches de vous</string>
     <string name="turn_gps_on">Activer votre GPS pour voir les événements proches de vous</string>
     <string name="activate_gps">ACTIVER GPS</string>
+    <string name="account_settings">Paramètres du compte</string>
+    <string name="accountEditScreenButton">Editer compte</string>
```
Comment (leagrieder) [inline app/src/main/res/values-fr/strings.xml:173]: "Modifier le compte" or "Modifier compte" would be better to my taste :)

**Review by leagrieder — APPROVED** [review]
Good PR! This was much needed. Now the top app bar is consistent across our app. I approve these changes

**Review by QuicheLorraine3600 — APPROVED** [review]
Simple, great and straightforward job, as every PR should be !

**Pull request merged**